modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -321,6 +347,9 @@
 	if(!cortical_owner.human_host)
 		to_chat(owner, span_warning("You must have a host to check blood!"))
 		return
+	var/obj/item/organ/internal/brain/victim_brain = cortical_owner.human_host.getorganslot(ORGAN_SLOT_BRAIN)
+	if(victim_brain)
+		cortical_owner.human_host.adjustOrganLoss(ORGAN_SLOT_BRAIN, 1)
```

Comment:
mm, I don't think doing brain damage is good for this one... maybe remove some blood?

Reply:
Set it to create a small blood drip. It's an easily-spammed ability that you might easily want to spam to keep an eye on host status - brain damage wasn't good for this one.